src/main/java/org/entando/kubernetes/client/core/DefaultEntandoCoreClient.java

```diff
@@ -630,4 +644,41 @@ private boolean isSafeDeleteResponseStatus(int status) {
                 || s.equals(HttpStatus.NOT_FOUND)
                 || s.equals(HttpStatus.FORBIDDEN));
     }
+
+    private boolean isRetryableResponseStatus(int status) {
+        List<HttpStatus> retryables = Arrays.asList(
```

Comment:
Make it static in order to avoid instantiating it at every execution

Reply:
done